python/cudf_polars/tests/experimental/test_parallel.py

```diff
@@ -19,3 +26,46 @@ def test_evaluate_dask():
     )
     assert_frame_equal(expected, got_gpu)
     assert_frame_equal(expected, got_dask)
+
+
+@pytest.mark.parametrize(
+    "agg",
+    [
+        pl.col("int").max(),
+        # Check LiteralColumn serialization
```

Comment:
Is this comment just supposed to mean that the test case below is used to check that `LiteralColumn` serialialization works?

Reply:
Yes, exactly. This is just a note (for future reference) that the next line was specifically for this purpose.